frontend/src/metabase/query_builder/components/view/ViewHeader/components/DashboardBackButton/DashboardBackButton.tsx

```diff
@@ -9,10 +10,11 @@ import { ActionIcon, type ActionIconProps, Icon, Tooltip } from "metabase/ui";
 
 import DashboardBackButtonS from "./DashboardBackButton.module.css";
 
-type DashboardBackButtonProps = {
+export type DashboardBackButtonProps = {
   noLink?: boolean;
   onClick?: () => void;
-} & ActionIconProps;
+} & ActionIconProps &
+  HTMLAttributes<HTMLButtonElement>;
```

Comment:
I think we haven't mentioned in the docs that these also support `HTMLAttributes<HTMLButtonElement>` alongside the ActionIconProps, but I think it's fine.